### PR TITLE
Update addon via `ember init`

### PR DESCRIPTION
**Pull request by elwayman02**

### CODE REVIEW

File: tests/unit/initializers/responsive-test.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* global sinon */
 import Ember from 'ember';
 import initializer from '../../../initializers/responsive';
-import instanceInitializer from '../../../instance-initializers/responsive';
```
Comment (alexBaizeau) [inline tests/unit/initializers/responsive-test.js]: Why is that not necessary anymore?

Comment (elwayman02) [inline tests/unit/initializers/responsive-test.js]: It wasn't being used anywhere in the file... Jshint threw an error.

**Comment by RuslanZavacky** [comment]
@elwayman02 any chance to update it for ember 2.1 and proceed with this PR? :)

**Comment by elwayman02** [comment]
@RuslanZavacky @alexBaizeau done.

**Comment by alexBaizeau** [comment]
Thanks a lot @elwayman02

**Pull request merged**